diff --git a/Library/Formula/python3.rb b/Library/Formula/python3.rb
index 4632e5a..d149635 100644
--- a/Library/Formula/python3.rb
+++ b/Library/Formula/python3.rb
@@ -11,7 +11,7 @@ class Python3 < Formula
     sha1 "f2df5040f65cd509bbd33666fe09eb329d2b81bc" => :mountain_lion
   end
 
-  head "https://hg.python.org/cpython", :using => :hg, :branch => "3.5"
+  head "https://hg.python.org/cpython", :using => :hg, :branch => "default"
 
   devel do
     url "https://www.python.org/ftp/python/3.5.0/Python-3.5.0a1.tgz"
@@ -142,7 +142,7 @@ class Python3 < Formula
     rm bin/"2to3"
 
     # Remove the site-packages that Python created in its Cellar.
-    site_packages_cellar.rmtree
+    site_packages_cellar.rmtree if site_packages_cellar.exist?
 
     %w[setuptools pip].each do |r|
       (libexec/r).install resource(r)
